bundle/DependencyInjection/Compiler/MaxUploadSizeParameterPass.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryFormsBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class MaxUploadSizeParameterPass implements CompilerPassInterface
+{
+    const MAX_UPLOAD_SIZE = 'ezplatform.env.max_upload_size';
```

Comment:
`ezplatform.env.max_upload_size` will be resolved on container build which in most cases will be resolved based on `php.ini` for CLI environment instead of apache/nginx specific once (upload_max_filesize may differ). 

Another side effect is that changing php.init setting require clearing cache which at least needs to be documented.

Reply:
Maybe then, we should just drop this max thing and just stick to setting min value (as the task originally says).